src/lib/utils/preconditions/conditions/PreconditionConditionAnd.ts

```diff
@@ -6,16 +6,16 @@ import type { IPreconditionCondition } from './IPreconditionCondition';
  * @since 1.0.0
  */
 export const PreconditionConditionAnd: IPreconditionCondition = {
-	async sequential(message, command, entries) {
+	async sequential(message, command, entries, context) {
 		for (const child of entries) {
-			const result = await child.run(message, command);
+			const result = await child.run(message, command, context);
```

Comment:
Isn't this added required parameter a breaking change? Please document it if so.